serf/serf.go

```diff
@@ -77,11 +77,13 @@ type Serf struct {
 	eventMinTime    LamportTime
 	eventLock       sync.RWMutex
 
-	queryBroadcasts *memberlist.TransmitLimitedQueue
-	queryBuffer     []*queries
-	queryMinTime    LamportTime
-	queryResponse   map[LamportTime]*QueryResponse
-	queryLock       sync.RWMutex
+	queryBroadcasts        *memberlist.TransmitLimitedQueue
+	queryBuffer            []*queries
+	queryMinTime           LamportTime
+	queryResponse          map[LamportTime]*QueryResponse
+	queryResponseSizeLimit int // Maximum bytes size for response
```

Comment:
I'm not sure this code will build as-is. This should be added to the Config structure, not the Serf object itself.